packages/firebird/src/_internal/data-types-overrides.ts

```
@@ -0,0 +1,175 @@
+
+// ── Binary ────────────────────────────────────────────────────────────────────
+
+export class BLOB extends BaseTypes.BLOB {
+  toSql(): string {
+    return 'BLOB';
+  }
+}
+
+// ── JSON ──────────────────────────────────────────────────────────────────────
+
+export class JSON extends BaseTypes.JSON {
```

Biome (2.4.6)

[error] Do not shadow the global "JSON" property.

(lint/suspicious/noShadowRestrictedNames)